ipn/ipnserver/server.go

```diff
@@ -304,8 +304,14 @@ func (s *Server) checkConnIdentityLocked(ci *ipnauth.ConnIdentity) error {
 			return inUseOtherUserError{fmt.Errorf("Tailscale already in use by %s, pid %d", active.User().Username, active.Pid())}
 		}
 	}
-	if cu := s.b.CurrentUser(); cu != "" && cu != ci.UserID() {
-		return inUseOtherUserError{fmt.Errorf("Tailscale already in use by %s", s.b.CurrentUser())}
+	if cu := s.b.CurrentUserID(); cu != "" && cu != ci.UserID() {
+		// TODO(maisem): cache this?
+		u, err := user.LookupId(cu)
```

Comment:
Should we use the `LookupUserByID` in ipnauth here?

Reply:
yeah probably

Reply:
Done